src/sentry/static/sentry/app/components/charts/releaseSeries.tsx

```diff
@@ -17,7 +18,7 @@ import {escape} from 'app/utils';
 import {getFormattedDate, getUtcDateString} from 'app/utils/dates';
 import {formatVersion} from 'app/utils/formatters';
 import parseLinkHeader from 'app/utils/parseLinkHeader';
-import theme from 'app/utils/theme';
+import theme, {Theme} from 'app/utils/theme';
```

Comment:
now that the component is wrapped with `withTheme`, should we remove this import?

Reply:
good catch